ttnn/cpp/ttnn/operations/eltwise/unary/device/unary_device_operation.cpp

```diff
@@ -30,10 +30,8 @@ void validate_supported_arch_dtype(
         case UnaryOpType::FILL:
            if(arch == tt::ARCH::GRAYSKULL){
                 TT_FATAL(
-                    input_datatype == DataType::BFLOAT16,
-                    "Supported input data type '{}' for UnaryOpType '{}' on Grayskull.",
-                    static_cast<int>(input_datatype),
-                    static_cast<int>(op_type));
+                    (input_datatype == DataType::BFLOAT16 || input_datatype == DataType::BFLOAT8_B),
+                    "Supported dtypes on Grayskull are BFLOAT16/BFLOAT8_B.");
```

Comment:
https://github.com/tenstorrent/tt-metal/blob/main/ErrorMessageBestPractices.md

"Unsupported dtype {}. On Grayskull only BFLOAT16/BFLOAT8_B are supported"

Inclusion of the actual type will be useful for users